administrator/components/com_menus/src/View/Menu/XmlView.php

```diff
@@ -134,7 +134,7 @@ protected function addXmlChild($xml, $item)
 			$node['element'] = $item->element;
 		}
 
-		if ($item->class)
+		if (isset($item->class))
```

Comment:
```suggestion
		if (isset($item->class) && $item->class)
```
Maybe we should do it like this to cater for empty string? `isset($item->class)` will return true for an empty string, but `$item->class` will resolve to false if empty string.

Reply:
For the other change below for `$item->submenu` we don't need that because that is not a string but an array. But here it could make sense.

Reply:
But maybe it's not necessary here either, and the PR is ok as it is. Opinions are welcome.

Reply:
sound better than before  to me